checks/http/sophos.rb

```diff
@@ -30,7 +30,7 @@ def generate_checks(url)
               vendor: 'Sophos',
               product: 'UTM',
               website: 'https://www.sophos.com/en-us/products/unified-threat-management.aspx',
-              description: 'devuce configuration - unique string',
+              description: 'device configuration - unique string',
```

Comment:
should it be "Device" with capital D or even even changing the description to be inline with the new checks?